app/matchers/bulkrax/application_matcher.rb

```diff
@@ -20,9 +20,9 @@ def result(_parser, content)
         return unless content.send(self.if[0], Regexp.new(self.if[1]))
       end
 
-      @result = content.to_s.gsub(/\s/, ' ') # remove any line feeds and tabs
-      @result.strip!
-      process_split
+      # @result will evaluate to an empty string for nil content values
+      @result = content.to_s.gsub(/\s/, ' ').strip # remove any line feeds and tabs
+      process_split if @result.present?
```

Comment:
now that `@result` can be an empty string (which is truthy in rails), we're checking for presence.